Miscellaneous-Server.md

```diff
@@ -317,61 +321,94 @@ WorkingDirectory=/path/to/topaz
 # - Uncomment and edit the 2 lines below with the appropriate port and log location for each zone server.
 #EnvironmentFile=/path/to/topaz/ip.txt
 #ExecStart=/path/to/topaz/topaz_game --ip $IP --port 54230 --log /path/to/log/map_server.log
-ExecStart=/path/to/topaz/topaz_game
+ExecStart=/path/to/lsb/xi_map
 
 [Install]
-WantedBy=topaz.service
+WantedBy=lsb.service
 ```
 
 ```ini
-#topaz_connect.service
+#lsb_connect.service
 
 [Unit]
-Description=Topaz Connect Server
+Description=LSB Connect Server
 Wants=network.target
 StartLimitIntervalSec=120
 StartLimitBurst=5
-PartOf=topaz.service
-After=topaz.service
+PartOf=lsb.service
+After=lsb.service
 
 [Service]
 Type=simple
 Restart=always
 RestartSec=5
-User=topaz
-Group=topaz
-WorkingDirectory=/path/to/topaz
-ExecStart=/path/to/topaz/topaz_connect
+# You can use your regular system user or a custom user you make
+# ie User=xiplayer or User=myregularusername
+User=
+# omit Group and it will use the Users primary group. Otherwise you can explicitly list it
+# Group=
+WorkingDirectory=/path/to/lsb
+ExecStart=/path/to/lsb/xi_connect
 
 [Install]
-WantedBy=topaz.service
+WantedBy=lsb.service
 ```
 
 ```ini
-#topaz_search.service
+#lsb_search.service
 
 [Unit]
-Description=Topaz Search Server
+Description=LSB Search Server
 Wants=network.target
 StartLimitIntervalSec=120
 StartLimitBurst=5
-PartOf=topaz.service
-After=topaz.service
+PartOf=lsb.service
+After=lsb.service
 
 [Service]
 Type=simple
 Restart=always
 RestartSec=5
-User=topaz
-Group=topaz
-WorkingDirectory=/path/to/topaz
-ExecStart=/path/to/topaz/topaz_search
+# You can use your regular system user or a custom user you make
+# ie User=xiplayer or User=myregularusername
+User=
```

Comment:
Does this require users to come in here and edit this, making this no longer completely copy/paste like it was before?

Reply:
My understanding was it still required someone to enter an appropriate User (and Group). The wiki had "topaz" as the value for User. If someone copy/pasted that then that person would have to make a User called topaz. 

Most people copy/pasting this should be fine using their normal User on their linux system...My username is grahf on my system and I use that, grahf, for the value of User. People can still make a new user on their system called "lsb" or whatever but I'm not sure what you would want to. Maybe there's a good reason but I don't know it.